README.md

```diff
@@ -60,7 +60,13 @@ GoRails should help you to get started on Ruby on Rails.
     $ bundle exec rake db:setup
     ~~~
 
- 6. Start [webpack](https://webpack.github.io/) and the development app server
+ 6. Generate translation strings for the frontend.
+
+    ~~~ sh
+    $ cd client && npm run extract-translations; cd -
```

Comment:
Actually `yarn build:development` already extracts the translations. It was mentioned in the section below `if you are not using foreman....`

Reply:
I ran the steps in a different order just now so thought it didn't work. Took it out again, but left the change removing `in beta`.